src/energyplus/ForwardTranslator/ForwardTranslateControllerOutdoorAir.cpp

```diff
@@ -151,6 +159,17 @@ namespace energyplus {
       }
     }
 
+    // HumidistatControlZoneName
+    if (auto zone_ = modelObject.humidistatControlZone()) {
+      if (boost::optional<ZoneControlHumidistat> humidistat = zone_->zoneControlHumidistat()) {
+        idfObject.setString(openstudio::Controller_OutdoorAirFields::HumidistatControlZoneName, zone_->nameString());
+      } else {
+        LOG(Warn, modelObject.briefDescription() << " has a humidistat control zone " << zone_->nameString()
+                                                 << " without a zone control humidistat; humidistat control zone field will not be translated");
+        idfObject.setString(openstudio::Controller_OutdoorAirFields::HighHumidityControl, "No");
```

Comment:
wait, didn't you say that the other two fields might be using this? I'm not familiar with that E+ functionality, mind you.

Reply:
Ok, I see https://bigladdersoftware.com/epx/docs/24-2/input-output-reference/group-controllers.html#field-high-humidity-control

So a ZOne with a humidistat is needed for High Humidity Control = Yes